notebooks/finetune/Flan-T5-3B/RosaQA.ipynb

```diff
@@ -0,0 +1,7957 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.    # trainable params: 18874368 || all params: 11154206720 || trainable%: 0.16921300163961817
</pre><p> </p><p>the output is shown below the cell, so maybe remove that comment here - especially because it differs from the actual output :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/foundation-models-for-documentation/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>